### PR TITLE
Fixes #650 Open multiple versions of the same panel

### LINKED ISSUE #650: Allow opening multiple versions of the same browser
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of the CDP4-IME
- [x] I have searched [open](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/issues) and [closed](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
Add the ability to open multiple versions of the same browser / panel.

### PR DESCRIPTION
**Pull request by lxatrhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/pulls) open
- [x] I have verified that I am following the CDP4-IME [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/CDP4-IME-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description
Open multiple versions of the same panel.

### CODE REVIEW

**Review by samatrhea — APPROVED** [review]

**Review by alexatrhea — APPROVED** [review]
Please bump composition version.

File: CDP4Addin/Addin.cs [hunk 1]
```diff
@@ -311,7 +315,7 @@ public override void CTPFactoryAvailable(object CTPFactoryInst)
         /// <param name="exception">The exception that occurred.</param>
         protected override void OnError(ErrorMethodKind methodKind, Exception exception)
         {
-            Utils.Dialog.ShowError(exception, "Unexpected state in CDP4-CE.Addin " + methodKind.ToString());
+            this.Utils.Dialog.ShowError(exception, "Unexpected state in CDP4-CE.Addin " + methodKind.ToString());
```
Comment (samatrhea) [inline CDP4Addin/Addin.cs:318]: why remove `this.`? i think this is a Utils instance from the base class coming from NetOffice, please revert

**Review by samatrhea — APPROVED** [review]
code looks good to me, would be good if this can be documented on the GH wiki, because now we have 2 distinct methods, once that is message based and another is not.

File: CDP4Addin/Addin.cs (hunk 1 above)
Comment (lxatrhea) [reply CDP4Addin/Addin.cs:318]: `this.` was not removed, but added. Probably by the beautifier.

Comment (samatrhea) [reply CDP4Addin/Addin.cs:318]: hahaha, good, even better, ignore my comment

**Review by nathanatrhea — APPROVED** [review]

**Review by ahmedatrhea — APPROVED** [review]

**Review by alexatrhea — APPROVED** [review]

**Pull request merged**